domains/auth/Auth.API/API/Utilities/Cookies.cs

```diff
@@ -0,0 +1,27 @@
+using API.Configuration;
+using Microsoft.Extensions.Options;
+
+
+namespace API.Services;
+
+public class Cookies : ICookies
```

Comment:
@C-Christiansen was the only usage of this utility in the tokencontroller to create the test endpoint? Or did i forget something?